tools/resources.py

```diff
@@ -56,7 +57,12 @@ def _plugin_path_dependency() -> str:
             top_level_name, *_ = module_name.split(".", maxsplit=1)
             top_level_module = sys.modules.get(top_level_name)
             if top_level_module is not None:
-                top_level_directory = Path(inspect.getfile(top_level_module)).parent
+                try:
+                    top_level_directory = Path(inspect.getfile(top_level_module)).parent
+                except TypeError:
+                    # TypeError gets thrown if
```

Comment:
inspect.get file raises TypeError if the object is a built-in module, class, or function. Could you please elaborate in the comment why it is safe to ignore these packages in this case.

Reply:
True, it is not that clear where this comes from.